### PR TITLE
Make metadata-cache able to start on boot as an init script.

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

File: jenkins/metadata-cache/metadata-cache-control.sh [hunk 1]
```diff
@@ -94,6 +105,10 @@ do_local() {
       ;;
     start)
       echo -n "Starting metadata-cache session: "
+      if [ -x "$(which metadata-cache.py)" ]; then
```
Comment (spxtr) [inline jenkins/metadata-cache/metadata-cache-control.sh]: Two square braces as per [the style guide](https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B).

Comment (fejta) [inline jenkins/metadata-cache/metadata-cache-control.sh]: Actually I think `$(which foo)` is fine?

File: jenkins/metadata-cache/README.md [hunk 2]
```diff
@@ -60,3 +60,12 @@ jenkins/metadata-cache/metadata-cache-control.sh remote_ssh $INSTANCE connect
 # Disable cache
 jenkins/metadata-cache/metadata-cache-control.sh remote_ssh $INSTANCE off
 ```
+
+### Running as a Service
+
+Copy metadata-cache-control.sh to /etc/init.d/metadata-cache, and
```
Comment (fejta) [inline jenkins/metadata-cache/README.md]: Please make the script do this automatically when running remote_update

Comment (rmmh) [inline jenkins/metadata-cache/README.md]: Done.

**Pull request merged**